test/verify/check-metrics

```diff
@@ -426,16 +429,8 @@ class TestHistoryMetrics(MachineCase):
         # first check the "no logs found" case
         login(self)
         b.wait_in_text(".metrics-heading", "CPU")
-        b.click("#metrics-hour-1615197600000 div.metrics-minute[data-minute=39] .metrics-events button")
-        b.wait_in_text("#metrics-hour-1615197600000 div.metrics-minute[data-minute=39] .cockpit-log-panel", "No logs found")
-        b.click("button:contains('View detailed logs')")
-        b.enter_page("/system/logs")
-        # jumps to the right time with larger window and debug level
-        url = b.eval_js('window.location.hash')
-        self.assertIn("priority=debug", url)
-        self.assertIn("since=2021-3-8%2010%3A24%3A0", url)
-        self.assertIn("until=2021-3-8%2010%3A40%3A0", url)
-        b.go("/metrics")
```

Comment:
The logs fetching is done automatically now for the expanded hours, and if there are no logs, there is not button. So all this is be now replaced with:

b.wait_in_text("#metrics-hour-1615197600000 div.metrics-minute[data-minute=39] .metrics-events span.spikes_info", "Load")
 
which is just the spike description span without being a button.
I could add a:

b.wait_not_present("#metrics-hour-1615197600000 div.metrics-minute[data-minute=39] .metrics-events button")

to make this button absense more explicit.

Reply:
@martinpitt I added the requested test coverage. Let me know if you still are worrying about the missing link to logs page, in case no logs are detected.